src/point_one/fusion_engine/messages/device.h

```diff
@@ -0,0 +1,44 @@
+/**************************************************************************/ /**
+ * @brief Device-specific messages.
+ * @file
+ ******************************************************************************/
+
+#pragma once
+
+#include "point_one/fusion_engine/common/portability.h"
+#include "point_one/fusion_engine/messages/defs.h"
+
+namespace point_one {
+namespace fusion_engine {
+namespace messages {
+
+// Enforce 4-byte alignment and packing of all data structures and values.
+// Floating point values are aligned on platforms that require it. This is done
+// with a combination of setting struct attributes, and manual alignment
+// within the definitions. See the "Message Packing" section of the README.
+#pragma pack(push, 1)
+
+struct alignas(4) SystemStatusMessage : public MessagePayload {
```

Comment:
Make sure to document this class, including brief with ref and version similar to `PoseMessage`, and include an `@ingroup` to add it to a doxygen message group.

I'd suggest we define a new message group `device_status` that falls under `@ingroup messages` see `solution.h` for an example.